WordPress/src/main/java/org/wordpress/android/ui/stories/StoryComposerActivity.kt

```diff
@@ -417,6 +416,16 @@ class StoryComposerActivity : ComposeLoopFrameActivity(),
     }
 
     override fun onSubmitButtonClicked(publishPost: PublishPost) {
+        prepareAndProcessStoryPostForPublishing()
+    }
+
+    private fun prepareAndProcessStoryPostForPublishing() {
+        editPostRepository.update { postModel ->
+            postUtilsWrapper.preparePostForPublish(
+                    postModel,
+                    requireNotNull(site)
+            )
+        }
         processStorySaving()
     }
 }
```

Comment:
Upon doing some more investigation I am realizing that this logic is not required because we do setup the post for publishing. @mzorz in the `StoryMediaSaveUploadBridge` you do this here 

https://github.com/wordpress-mobile/WordPress-Android/blob/bbc3c278385c049ebb6b49216fc2d40574d6143c/WordPress/src/main/java/org/wordpress/android/ui/stories/StoryMediaSaveUploadBridge.kt#L98

So yes, the `Published` `PostStatus` gets added there and that's actually the right place to do it, so I am thankful that I spotted it :)